config/delta-producer/public/export.json

```diff
@@ -80,14 +82,14 @@
     {
       "type": "http://www.w3.org/ns/locn#Address",
       "graphsFilter": [
-        "http://mu.semte.ch/graphs/organisatieportaal"
+        "http://mu.semte.ch/graphs/worship-service",
```

Comment:
Same, no shared graph ?

Reply:
they shouldn't be in the shared graph i think